api/src/main/java/org/openmrs/module/patientflags/api/impl/FlagServiceImpl.java

```diff
@@ -243,7 +269,20 @@ public void purgeFlag(Integer flagId) {
 	public Tag getTag(Integer tagId) {
 		return dao.getTag(tagId);
 	}
-	
+
+	/**
+	 * @see org.openmrs.module.patientflags.api.FlagService#getTagByUuid(String)
+	 */
+	public Tag getTagByUuid(String uuid) {
+		return dao.getTagByUuid(uuid);
+	}

```

Comment:
You need to have at least a line between methods.

Reply:
Done